server/sonar-web/src/main/js/apps/permission-templates/views/UsersView.js

```diff
@@ -62,9 +85,10 @@ export default Modal.extend({
   },
 
   serializeData () {
-    return _.extend(Modal.prototype.serializeData.apply(this, arguments), {
-      permissionName: this.options.permission.name,
+    return {
+      ...Modal.prototype.serializeData.apply(this, arguments),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use the rest syntax to declare this function's arguments. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3513)